.gitignore

```diff
@@ -1,4 +1,5 @@
 vendor/
 bin/
 composer.lock
-.idea
\ No newline at end of file
+.idea
+nbproject
```

Comment:
Those two lines should be removed becuase we can't support all IDEs. You can add those lines into your globla .gitignore. 
https://help.github.com/articles/ignoring-files/